src/plugins/data_view_editor/public/components/data_view_editor_flyout_content.tsx

```diff
@@ -100,7 +123,15 @@ const IndexPatternEditorFlyoutContentComponent = ({
         };
       }
 
-      await onSave(indexPatternStub);
+      if (
+        editData &&
+        editData.title !== form.getFields().title.value &&
```

Comment:
Just want to confirm that this is expected that we warn only about `title` change but not name or time field change

Reply:
Yes this is expected.